### ISSUE TITLE
C0072: Operation '__Delete' is not possible on type 'THIS'

### ISSUE THREAD

**Issue by piratecarrot** [Mobject-Dev-Team/mobject-disposable#4]
TwinCAT 3.1.4026.5

```
Error C0072: Operation '__Delete' is not possible on type 'THIS'	mobject-disposable (mobject-disposable\mobject-disposable)	C:\xxx\Workspaces\mobject\mobject-disposable\src\mobject-disposable-library\mobject-disposable\mobject-disposable\Disposable\Disposable.TcPOU@Dispose (Impl)	2	
```

Proposed fix
```
METHOD PUBLIC Dispose
VAR
	pThis : POINTER TO Disposable;
END_VAR

pThis := THIS;
__DELETE(pThis);
```

**Comment by benhar-dev** [comment]
Hi @piratecarrot, I have a new update to most libs coming shortly, so I'll add this in at the same time.  Thank you.

**Comment by benhar-dev** [comment]
Resolved in v1.1.0 👍
